moduledata.go

```diff
@@ -55,6 +55,8 @@ type Moduledata interface {
 	Bss() ModuleDataSection
 	// NoPtrBss returns the noptrbss section.
 	NoPtrBss() ModuleDataSection
+	// Covctrs returns the covctrs section.
+	CovcTrs() ModuleDataSection
```

Comment:
Do you know of a need to expose this data since it was added to 1.20? If not, let's not expose it.